Tests/MongoSwiftSyncTests/UnifiedTestRunner/ExpectedEventsForClient.swift

```diff
@@ -100,5 +112,8 @@ enum ExpectedEvent: Decodable {
 
         /// Specifies whether the serviceId field of the event is set.
         let hasServiceId: Bool?
+
+        /// Specifies whether the `serviceConnectionId` field of an event is set.
+        let hasServerConnectionId: Bool?
```

Comment:
Can we add a TODO comment for SWIFT-1262 to update the runner to use this value? That way we'll remember to update this if necessary, and also if someone comes across this field in the future they'll know why it's just here for deserialization.

Reply:
done